src/scythe-bidder/create-room.tsx

```diff
@@ -4,29 +4,33 @@ import React from "react";
 import { jsx } from "@emotion/core";
 import { Button, Card, notification, Select, Form } from "antd";
 import client from "./client";
-import { MAX_PLAYERS, MIN_PLAYERS, SCYTHE_BIDDER } from "./constants";
+import { MAX_PLAYERS, MIN_PLAYERS, SCYTHE_BASE, SCYTHE_IFA, MAX_PLAYERS_BASE, MAX_PLAYERS_IFA } from "./constants";
 
 export default function CreateRoom({ onCreate }: { onCreate: () => void }) {
   const [numPlayers, setNumPlayers] = React.useState(2);
+  const [gameType, setGameType] = React.useState<string>(SCYTHE_BASE);
```

Comment:
If we are doing just IFA in this PR, it's a binary choice that can be represented by a boolean, right? I'm thinking instead of using the Select UI component, you could use a [Switch](https://ant.design/components/switch/) component instead.

Reply:
Actually, we need this to (1) select the available mats & factions and (2) set the max number of players.

Reply:
I am comfortable changing it to use a Switch (and it looks better too). But if we want to implement mat filters or tier filters or anything else, it will no longer be a binary choice and we will need to change it to something else. I'm happy to change it to use a number if you think that's better.

Reply:
What do you think? Do you want to turn this pull request into being able to completely customize the combinations used? I'm thinking of having some grid of checkboxes and then presets like "No IFA" or "No SS tier" to minimize most of the work... The issue with this that I can think of right now is that it's possibly a tricky algorithm problem to assign the combinations and also to know whether inputs are valid given the number of players.

Reply:
No, I don't want to complicate things right at the start. Just the IFA toggle for now.

For the tier / faction / mat customization, I can see a bit-wise variable (value set from check boxes / toggles / ?) being passed to setupData.